ext/random/tests/03_randomizer/basic.phpt

```diff
@@ -27,6 +27,19 @@ $engines[] = new UserEngine();
 foreach ($engines as $engine) {
     $randomizer = new Random\Randomizer($engine);
 
+    // nextInt
+    for ($i = 0; $i < 1000; $i++) {
+        try {
+            if (!\is_int($randomizer->nextInt())) {
```

Comment:
The result cannot be not int due to the return type

Reply:
fixed in new PR: https://github.com/php/php-src/pull/9057